pkg/azurediskplugin/main.go

```diff
@@ -37,9 +41,12 @@ func init() {
 }
 
 var (
-	endpoint                   = flag.String("endpoint", "unix://tmp/csi.sock", "CSI endpoint")
+	driverConfig azdiskv1beta2.AzDiskDriverConfiguration
+	driverConfigPath           = flag.String("config", "", "The configuration path for the driver")
```

Comment:
It would be good to create constants for the config & parameters default values in `pkg/azureconstants`.